packages/frontend-v2/components/Form/Input.tsx

```diff
@@ -0,0 +1,33 @@
+import {
+  FormControl,
+  Input,
+  FormErrorMessage,
+  InputGroup,
+} from "@chakra-ui/react";
+import { FieldError } from "react-hook-form";
+import { forwardRef } from "react";
+
+interface InputProps {
+  error: FieldError | undefined;
+  id: string;
+  placeholder: string;
+  type: string;
+}
+
+// eslint-disable-next-line react/display-name
```

Comment:
forward refs are weird with this. Not sure how to bypass without adding this line